### DISCUSSION TITLE
I cannot submit bug report

### DISCUSSION THREAD

**Discussion by jdgamble555** [vercel/community#5504] [Help] [answered]
![Screenshot 2024-01-17 105236](https://github.com/vercel/community/assets/8333090/83e2dd32-8964-4d8e-9f44-34af82ddaa98)

I login, there is no place to submit a bug report:
```
https://vercel.com/support/request
```
So I am posting here.  Huge problem.

J

**Comment by swarnava** [comment]
Hi

What is the issue are you facing?

**Comment by jdgamble555** [comment]
That I can't report a bug lol. It is a bug.

**Reply by swarnava** [reply, marked as answer]
You can contact Vercel Support using https://vercel.com/help
